src/measure_theory/probability_mass_function.lean

```diff
@@ -157,22 +157,22 @@ def normalize (f : α → ℝ≥0) (hf0 : tsum f ≠ 0) : pmf α :=
   (mul_inv_cancel hf0) ▸ has_sum.mul_right (∑' x, f x)⁻¹
     (not_not.mp (mt tsum_eq_zero_of_not_summable hf0 : ¬¬summable f)).has_sum⟩
 
-lemma normalize_apply (f : α → ℝ≥0) {hf0 : tsum f ≠ 0} {a : α} :
+lemma normalize_apply (f : α → ℝ≥0) (hf0 : tsum f ≠ 0) (a : α) :
   (normalize f hf0) a = f a * (∑' x, f x)⁻¹ := rfl
 
 /-- Create new `pmf` by filtering on a set with non-zero measure and normalizing -/
 def filter (p : pmf α) (s : set α) (h : ∃ a ∈ s, p a ≠ 0) : pmf α :=
 pmf.normalize (s.indicator p) $ nnreal.tsum_indicator_ne_zero p.2.summable h
 
-lemma filter_apply (p : pmf α) {s : set α} {h : ∃ a ∈ s, p a ≠ 0} {a : α} :
+lemma filter_apply (p : pmf α) (s : set α) (h : ∃ a ∈ s, p a ≠ 0) {a : α} :
```

Comment:
`s` appears in `h` so it can be inferred. Same in the other lemmas below.

(`p` can be inferred too, but it's better to leave them explicit so that we can use dot notation.)